### ISSUE TITLE
Support Fallback Focus in Terra Abstract Modal

### ISSUE THREAD

**Issue by papasmile** [cerner/terra-core#2069]
# Bug Report

## Description
Focus Trap as of 3.0.0[1] requires at least one focusable element or it fails with error:  "Error: You can't have a focus-trap without at least one focusable element"

## Steps to Reproduce

Example code:
```
<AbstractModal
  isOpen={state.isOpenModal}
  onRequestClose={closeModalFunc}
  ariaLabel="Hi"
>
  Have a nice day.  Click anywhere to continue
</AbstractModal>

When modal attempts rendered, error occurs.

## Additional Context / Screenshots
There was a related change which removes modal from tabs only on iOS: https://github.com/cerner/terra-core/pull/2017 

## Expected Behavior
<!-- A clear and concise description of what you expected to happen. -->

## Possible Solution
* Set 'fallbackFocus' to modal container
* OR expose 'fallbackFocus' to clients
* OR next another div with tabindex=0 (if that plays ok with accessibility)

## Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
* Component Name and Version: Terra Abstract Modal >= 1.20
* Browser Name and Version: iOS => 11.2
* Node/npm Version: [e.g. Node 8/npm 5] Node 8.9.2 / Npm 5.5.1
* Webpack Version: Webpack 4.26.0
* Operating System and version (desktop or mobile):

## @ Mentions
@bjankord via Teams

**Comment by papasmile** [comment]
Also could be solve by just documenting focusable content as a requirement.

**Comment by JakeLaCombe** [comment]
I lean towards a mixture of option 1 and 2
